app/styles/_forms.scss

```diff
@@ -46,7 +46,7 @@ form {
   input,
   .ember-power-select-trigger,
   .form-dropdown-trigger {
-    line-height: $form-control-height - 4; // 1px border, 1px padding
+    line-height: $form-control-height - 2; // 1px border, 1px padding
```

Comment:
This styling applies to forms in the profile building pages as well, and was set up to make sure our various types of controls are the same height. `$form-control-height` is the overall height that we make all the form controls, so this line was subtracting 4 because of 1px of padding on the top, 1px on the bottom, 1px of border on the top and 1px on the bottom. I'm worried that by making this change, then these controls won't match other form controls in height, for example on the transportation page the input might not match the height of the checkbox-like controls.

Is this change specifically meant to address the login page and related modals? If so, it should probably be done in the login-specific SCSS, rather than in this file that does across-the-app form styling.

Reply:
I will keep this styling as original(-4) since I got your point for the check-box controls and the height of the submit button will have no big difference for making it to -2. Thanks for the clarification.